src/Fixer/Basic/BracesFixer.php

```diff
@@ -179,17 +179,22 @@ private function fixCommentBeforeBrace(Tokens $tokens)
             }
 
             $tokenTmp = $tokens[$braceIndex];
-            $trimIndex = $tokens->getPrevNonWhitespace($braceIndex);
-            $tokens[$trimIndex]->setContent(rtrim($tokens[$trimIndex]->getContent()));
```

Comment:
my guess is this is legacy code from 1.x branch when comments still had a trailing line break, as opposed to on 2.x where these are now two tokens